cli/cmd/delete_project.go

```diff
@@ -30,7 +30,7 @@ Example:
 
 		if len(args) != 1 {
 			cmd.SilenceUsage = false
-			return errors.New("Requires PROJECTNAME")
+			return errors.New(`required argument PROJECTNAME not set`)
```

Comment:
Is it neccessary to use backticks/backquotes here?

Reply:
No, it is not necessary here, but it does not make a difference.